src/start.c

```diff
@@ -164,7 +164,7 @@ void start_cleanup (void)
  */
 const char* start_name (void)
 {
-   return start_data.name;
+   return _(start_data.name);
```

Comment:
I'm not quite sure the function should return a translated text because it's saved the version.data in the save files.

Reply:
It should be not. It might be best to have all the `start_` functions be untranslated, and wrap them in `_()` as necessary.